input/pages/measure-conformance.md

```diff
@@ -863,7 +863,7 @@ The referenced CQL expressions return either an indication that a patient meets
 
 In Table 3-2, the first measure is an example of a patient-based measure. Each patient may contribute at most one count to the denominator and numerator, regardless of how many encounters they had. The second measure is an episode-of-care measure where each patient may contribute zero or more encounters to the denominator and numerator counts.
 
-For complete examples of patient-based proportion measures, see the Screening Measure [Examples](examples.html). For a complete example of an encounter-based proportion measure, see the [EXM108](Measure-EXM108-FHIR.html) measure included in this implementation guide.
+For complete examples of non-patient based proportion measures, see the Screening Measure [Examples](examples.html). For a complete example of an encounter-based proportion measure, see the [EXM108](Measure-EXM108-FHIR.html) measure included in this implementation guide.
```

Comment:
1) do not change "patient-based" to "non-patient based"
2) The resolution says "Throughout QMs section, change all instances of 'episode-of-care' or 'encounter-based' to 'non-patient based'
- To apply the changes according to this resolution, it needs to search the entire "measure-conformance.md" file and change all instances of "episode-of-care" based measures and "encounter-based" measures to "non-patient based" measures.